examples/5-cross-compiling-mingw-w64/Cargo.toml

```diff
@@ -1,8 +1,9 @@
 [package]
 name = "hello-world"
 version = "0.1.0"
-authors = ["Eyal Kalderon <eyal.kalderon@tenx.tech>"]
+authors = ["gjz010 <gjz010@gjz010.com>"]
```

Comment:
Author is not changed.

Reply:
It is a brand new test case that I added to test calling Win32 API through `winapi` and `windows_rs` crates. I will revert the change if you prefer that.

Reply:
Oh never mind.  I must have looked at the individual commit and not the whole change set.

Thanks for changes.  I'm currently back in the thick of deployment work and likely to do some overdue work here, including merging changes like these.